### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by brunogaspar**
Signed-off-by: Bruno Gaspar <brunofgaspar1@gmail.com>

### CODE REVIEW

**Comment by Niek** [comment]
Maybe add some instructions to get the versions: ``npm list puppeteer puppeteer-extra`` (possibly add `-json`). For Node: `node -v`.

**Comment by brunogaspar** [comment]
Good idea @Niek

**Comment by berstend** [comment]
Maybe not for this PR but we definitely need to mention the necessity of pasting relevant debug output (there's a bunch of recent "help recaptcha isnt' working" issues with no relevant debug info).

**Comment by brunogaspar** [comment]
I've updated it a little bit and also added a tweaked version of @Niek suggestion.

What should we add more?

Also should we consider tweaking the GitHub labels?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Questions & Support
+    url: https://discord.gg/MUSdsS
+    about: We don't offer official support - only community support. Join our 24/7 Discord server and post your question there, we've a great community growing, that are happy to help.
```
Comment (brunogaspar) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Can be worded differently and better, please leave your suggestions :)

**Comment by berstend** [comment]
I had some downtime and have it a shot: https://github.com/berstend/puppeteer-extra/commit/8d10f555c0af4ea206c9bab4439e93ccbba8dc40

Closing this PR as it's obsolete now. Thanks!

**Pull request closed without merging**